.github/workflows/cicd_pipeline.yml

```diff
@@ -25,27 +25,46 @@ jobs:
   build_bundle:
     name: "Build JS Bundle"
     if: github.event_name == 'push' || github.event.pull_request.draft == false
-    uses: heartexlabs/label-studio-frontend/.github/workflows/build_bundle.yml@master
+    uses: heartexlabs/label-studio-frontend/.github/workflows/build_bundle.yml@dev-3947/e2e-coverage
     with:
       sha: ${{ github.event.pull_request.head.sha || github.event.after }}
 
+  build_for_coverage:
+    name: "Build JS Bundle"
```

Comment:
Use another name

Reply:
it's different on the job level, here name is used as a group, so there'll be two jobs:
- Build JS Bundle / Build
- Build JS Bundle / Build Coverage